### DISCUSSION TITLE
Brightness sensor issue

### DISCUSSION THREAD

**Discussion by umbiAtkos** [MonitorControl/MonitorControl#980] [General]
Hi, 
I am currently using a 27 inch led Cinema Display connected to a Mac mini via an usb c to mini DP adapter and It seems like it is no longer possible to control whether having the auto-dimming function switched on or off, as the option is no longer available In system preferences. I've already set up the max DDC to 255 as mentioned, but it does not seem to change anything. I noticed that if I try to shine a light in front of the light sensor, it brings the brightness back to its full range and this is the only way I managed to achieve that on my machine... as a result I have zero control over the display brightness except for the software dimming option offered by the app. I was simply wondering if it was possible to add a feature that allows to turn off the sensor action in order to manually adjust brightness. Thanks in advance!

**Comment by waydabber** [comment]
Hi @umbiAtkos - I am not sure if this can be done via issuing a DDC command to the display. Normally Cinema Displays were controlled by Apple's proprietary control mechanism via USB commands (not DDC but apparently at least some firmware versions do accept DDC as well) but they removed these from macOS as part of some kind of planned obsolescence program. Some research would be needed to figure out how to handle this which would require hands-on with a display like this + an older mac that was able to control it originally?
